oblt-cli/setup/download.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+BIN_DIR="$HOME/oblt-cli/bin"
+
+mkdir -p "${BIN_DIR}"
+
+if [[ ${RUNNER_OS} == "Linux" ]]; then
+  if [[ ${RUNNER_ARCH} == "X64" ]]; then
+    PATTERN='*linux_amd64.tar.gz'
+  elif [[ ${RUNNER_ARCH} == "ARM64" ]]; then
+    PATTERN='*linux_arm64.tar.gz'
+  else
+    echo "Unsupported architecture for ${RUNNER_OS}: ${RUNNER_ARCH}"
+    exit 1
+  fi
+elif [[ ${RUNNER_OS} == "macOS" ]]; then
+  if [[ ${RUNNER_ARCH} == "X64" ]]; then
+      PATTERN='*darwin_amd64.tar.gz'
+    elif [[ ${RUNNER_ARCH} == "ARM64" ]]; then
+      PATTERN='*darwin_arm64.tar.gz'
+    else
+      echo "Unsupported architecture for ${RUNNER_OS}: ${RUNNER_ARCH}"
+      exit 1
+    fi
+else
+  echo "Unsupported OS: ${RUNNER_OS}"
+  exit 1
+fi
+
+if [ "${OBLT_CLI_VERSION}" == "6.5.3" ]; then
```

Comment:
I'd say this will be not needed. We can say this action will be only available when support the Keyless/Google Auth approach, what do you think?

Reply:
wait... this is just a warning